### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Blob Container Table_Screen Reader: Voice Over is not announcing the sorting information present for the column headers.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7594]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Regression From

NO

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#AILimited;#MAC;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#ScreenReader;#Voiceover;#WCAG1.3.1;#Element:Table;#Info and Relationships;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Turn on Voice Over.
2. Launch Storage Explorer.
3. Tab till "Open Connect dialog" pane and Sign in to Azure.
4. Tab till "Manage Accounts" pane and Select the subscriptions you will use. Once selected, hit enter on "Apply" button.
5. Tab till "Blob Containers" of the selected subscription, press context menu and hit enter on "Create Blob container" to Create a new Blob container.
6. Once completed, tab till "Upload" button and Upload a blob/file by using "Upload Files" control.
7. Tab till Table present in right side of the storage explorer.
8. Verify that Voice Over is announcing the sorting information present for the column headers or not.

### Actual Experience

On performing sorted functionality in column headers screen reader is not announce the sorting information like "sorted ascending" or "sorted descending".

### Expected Experience

On performing sorted functionality in column headers screen reader should announce the sorting information like "sorted ascending" or "sorted descending". Also when we navigate to the particular column header screen reader should announce the current sorting state.

### User Impact:
Screen Reader users will miss the sorting functionality if screen reader is not announcing the sorting information.

### Attachment:
![A11y_Azure Tools Storage Explorer_Blob Container Table_Screen Reader](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/9fe7fc91-43c4-428b-91ce-92685befc3ac)

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Version: 1.33.0 (97)
AzCopy Version: 10.23.0
Platform: macOS
Architecture: arm64
Build Number: 20240220.6
Issue is fixed. Hence Closing the bug. Please find added attachment for reference.
<img width="1440" alt="7594_Fixed" src="https://github.com/microsoft/AzureStorageExplorer/assets/98530069/82977d0a-a7fc-438d-a8b8-3fbeafcc35d0">

**Comment by keerthiduvvuri** [comment]
##Closed;
